### ISSUE TITLE
regression_loss not decrease 

### ISSUE THREAD

**Issue by kmktu** [zylo117/Yet-Another-EfficientDet-Pytorch#676]
Thank you your repo

i have some problem

my train command is
-c 0 --head_only False --batch_size 32 --lr 1e-4 --num_epochs 200 --load_weights weights/efficientdet-d0.pth

dataset train 6800, val 1700 images

first classification train and classification val loss decrease well
and total train loss and total val loss decrease well

but regression_loss decrease 10K then after 10k increase 

to solve increase lr and anchor size modify to https://github.com/mnslarcher/kmeans-anchors-ratios
after 10k increase lr re learning ...

I don't know why this is happening.

![image](https://user-images.githubusercontent.com/42626963/126753161-aec3533b-e6f6-4540-8e08-5a4db0495be4.png)

help me plz

**Comment by zylo117** [comment]
loss means nothing, you should check the mAP

**Comment by kmktu** [comment]
thanks for your reply

i check the mAP

![image](https://user-images.githubusercontent.com/42626963/128114151-c64ff508-0d54-4f05-b3f3-9a8090469052.png)

but not good result

i think my dataset is to be difficult so i change the model
